src/sonic-config-engine/tests/multi_npu_data/py3/ipinip.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
Why do we have this file created here? Is it new?

Reply:
This is an issue with git diff when viewed all at once. The respective commit will show the history and that it is moved. This why, I'd avoid squash merge